src/lib.rs

```diff
@@ -113,14 +113,14 @@ impl ManageConnection for RusqliteConnectionManager {
 
     async fn is_valid(
         &self,
-        conn: &mut bb8::PooledConnection<'_, Self>,
+        conn: &mut Self::Connection,
     ) -> Result<(), Self::Error> {
         // Matching bb8-postgres, we'll try to run a trivial query here. Using
         // block_in_place() gives better behaviour if the SQLite call blocks for
         // some reason, but means that we depend on the tokio multi-threaded
         // runtime being active. (We can't use spawn_blocking() here because
         // Connection isn't Sync.)
-        tokio::task::block_in_place(|| conn.execute("SELECT 1", NO_PARAMS))?;
+        tokio::task::block_in_place(|| conn.execute("SELECT 1", []))?;
```

Comment:
I stumbled on an issue with this line of code where `execute` will return an `Err(rusqlite::Error)` if anything being executed returns a row. I believe `execute_batch` fixes this. I've just removed the check entirely, I'm not sure it's necessary.

Reply:
just noting that `NO_PARAMS` was removed in https://github.com/rusqlite/rusqlite/releases/tag/v0.29.0 https://github.com/rusqlite/rusqlite/commit/d79d4a17